### PR TITLE
Site Settings: Remove Disabled Input for WordPress Version

### PR DESCRIPTION
**Pull request by Aurorum**
#### Changes proposed in this Pull Request

* Removes the disabled input in Site Settings for the WordPress version

> Some of the elements to mention should obviously not be a text input (why would the version number be a text input?)

#### Testing instructions

On a Jetpack connected site, visit `/settings/general` and check the WordPress version.

**Current:**

Screenshot 2019-04-21 at 14 03 43

**Proposed:**

Screenshot 2019-04-21 at 14 03 21

cc @simison, @drw158 

See #31207 and #26982 for some context on this

### CODE REVIEW

File: client/my-sites/site-settings/form-general.jsx [hunk 1]
```diff
@@ -129,15 +129,16 @@ export class SiteSettingsFormGeneral extends Component {
...
-					name="wpversion"
-					id="wpversion"
-					data-tip-target="site-title-input"
-					type="text"
-					value={ get( selectedSite, 'options.software_version' ) }
-					disabled
-				/>
+				<FormLabel 
+					className="site-settings__wordpress-version" 
+					htmlFor="wpversion"
```
Comment (ryelle) [inline client/my-sites/site-settings/form-general.jsx]: Can this be changed to a different component? ie, not `FormLabel` – there is no input with ID `wpversion` for this to label anymore (and if somehow there is, in some other component on the page, it shouldn't attach to this label)

Comment (torres126) [reply client/my-sites/site-settings/form-general.jsx]: 👍 

Updated the PR to just use a `<strong>` tag, would that be okay?

File: client/my-sites/site-settings/style.scss [hunk 2]
```diff
@@ -479,6 +479,11 @@ button.site-settings__general-settings-set-guessed-timezone.button.is-borderless
 		box-shadow: none;
 	}
 }
+
+.site-settings .site-settings__wordpress-version {
+	display: inline;
```
Comment (simison) [inline client/my-sites/site-settings/style.scss]: This probably isn't needed for `strong` tag anymore?

File: client/my-sites/site-settings/form-general.jsx (hunk 1 above)
Comment (simison) [reply client/my-sites/site-settings/form-general.jsx]: Better now! :-) There's also `FormFieldset` component which isn't semantically correct now.

File: client/my-sites/site-settings/style.scss (hunk 2 above)
Comment (torres126) [reply client/my-sites/site-settings/style.scss]: Yep, good call!

File: client/my-sites/site-settings/form-general.jsx (hunk 1 above)
Comment (torres126) [reply client/my-sites/site-settings/form-general.jsx]: I updated it to use Fragment. :)

**Review by lancewillett — CHANGES_REQUESTED** [review]
The UI changes look good to me, as well as the code changes.

Before:
Screen Shot 2020-05-05 at 12 16 37

After:
Screen Shot 2020-05-05 at 12 23 48

Change requested — please rebase, so I can use your branch to commit. 😄

**Comment by Aurorum** [comment]
Done, thank you! :)

**Review by lancewillett — APPROVED** [review]

**Pull request merged**